fcl/reco/Stage0/CMakeLists.txt

```diff
@@ -2,3 +2,4 @@ cet_enable_asserts()
 
 add_subdirectory(Run1)
 add_subdirectory(Run2)
+add_subdirectory(RunA)
```

Comment:
Fair enough on the name. Run 0 is already taken. Maybe PreRun1 would be a slight preference for me than RunA, but I'm also not sure I feel very strongly about it. If I think of something better I'll let you know.